egs/swbd/s5c/local/pytorchnn/data_prep.sh

```diff
@@ -33,15 +33,16 @@ if [ $# != 1 ]; then
 fi
 
 dir=$1 # data/pytorchnn/
+mkdir -p $dir
 
 for f in $train $valid $test $fisher; do
     [ ! -f $f ] && echo "$0: expected file $f to exist." && exit 1
 done
 
 # Sort and preprocess SWBD dataset
-local/pytorchnn/sort_by_start_time.py --infile $train --outfile $dir/swbd.train.sorted.txt
-local/pytorchnn/sort_by_start_time.py --infile $valid --outfile $dir/swbd.valid.sorted.txt
-local/pytorchnn/sort_by_start_time.py --infile $test --outfile $dir/swbd.test.sorted.txt
+python local/pytorchnn/sort_by_start_time.py --infile $train --outfile $dir/swbd.train.sorted.txt
```

Comment:
I prefer to always explicitly invoke python3, in case python2 is the default python.

Reply:
Sure. I will modify all related parts accordingly.